app/index.js

```diff
@@ -107,6 +111,19 @@ var NodeModuleGenerator = yeoman.Base.extend({
                 type: 'input',
                 name: 'username',
                 message: 'What is your GitHub username?',
+                store: true,
+                validate: function(input) {
+                    var res =   !input.startsWith('_')   &&
```

Comment:
I also wanted a single line b/c the &&'s alignment wasn't consistent w/ the rest of the code. (also my bad, that should have been a linter rule)